build/makelib/golang.mk

```diff
@@ -131,6 +134,14 @@ go.build: go.vet go.fmt $(CGO_PREREQS)
 	@$(foreach p,$(GO_NONSTATIC_PIE_PACKAGES),CGO_ENABLED=1 $(GO) build -v -i -o $(GO_OUT_DIR)/$(lastword $(subst /, ,$(p)))$(GO_OUT_EXT) $(GO_NONSTATIC_PIE_FLAGS) $(p);)
 	@$(foreach p,$(GO_STATIC_PACKAGES),CGO_ENABLED=0 $(GO) build -v -i -o $(GO_OUT_DIR)/$(lastword $(subst /, ,$(p)))$(GO_OUT_EXT) $(GO_STATIC_FLAGS) $(p);)
 	@$(foreach p,$(GO_STATIC_CGO_PACKAGES),CGO_ENABLED=1 $(GO) build -v -i -o $(GO_OUT_DIR)/$(lastword $(subst /, ,$(p)))$(GO_OUT_EXT) $(GO_STATIC_CGO_FLAGS) $(p);)
+ifeq ($(RELEASEBUILD),1)
+ifeq ($(GOOS),linux)
+#	@$(foreach p,$(GO_STATIC_CGO_PACKAGES) $(GO_NONSTATIC_PACKAGES) $(GO_NONSTATIC_PIE_PACKAGES),dwz $(GO_OUT_DIR)/$(lastword $(subst /, ,$(p)))$(GO_OUT_EXT);)
```

Comment:
`dwz` cleans up the DWARF symbols and shaves off another 100MB of the 700MB .debug file. It seems to have issues though due to a binutils bug which would require us to upgrade ubuntu in the build container. I left it here so that I can remember it.